discover/portal/articles/100-web-experience-management/03-staging-content-for-publication/03-using-the-staging-environment.markdown

```diff
@@ -121,6 +121,10 @@ the Web Content label. Otherwise, the Web Content section is absent.
 *Categories* and *Page Ratings* content types are not dependent on the date
 range, and are always shown in the list.
 
+| **Note:** Since some content types are meant for the end user (e.g. comments,
+| ratings, and custom fields), they are not supported in staging and can only be
+| added to the live site.
+
```

Comment:
Hey @Claudioss23 , same thing as I said [here](https://github.com/abhnerramos/liferay-docs/pull/11#discussion_r1435133494). But in this case, we should only be talking about the custom field, as the ticket only mentions the custom field.

So it would be something simple like `Note: Once published, custom fields do not get deleted in remote live.`

Reply:
Replied [here](https://github.com/abhnerramos/liferay-docs/pull/11#discussion_r1437583597).

I added the same note because, in this case, it works for both versions.